sdk/core/azure-core/azure/core/pipeline/policies/universal.py

```diff
@@ -278,17 +280,24 @@ class ContentDecodePolicy(SansIOHTTPPolicy):
     CONTEXT_NAME = "deserialized_data"
 
     @classmethod
-    def deserialize_from_text(cls, response, content_type=None):
-        # type: (Type[ContentDecodePolicyType], HTTPResponseType, Optional[str]) -> Any
+    def deserialize_from_text(
+        cls,  # type: Type[ContentDecodePolicyType]
+        data,  # type: Optional[Union[AnyStr, IO]]
+        content_type=None,  # Optional[str]
+        response=None  # Optional[_HttpResponseBase]
```

Comment:
FWIW it's a little odd that a documented type in a public API is a private class

Reply:
I could do `Union[HttpResponse, AsyncHttpResponse]` instead. More verbose, but at least not a private type, I agree

Reply:
fixed